src/main/scala/com/adobe/api/platform/runtime/mesos/MesosClient.scala

```diff
@@ -135,8 +137,10 @@ case class DeletePending(taskId: String, promise: Promise[Deleted]) extends Task
 case class Deleted(taskId: String, taskStatus: TaskStatus) extends TaskState
 case class Failed(taskId: String, agentId: String) extends TaskState
 
-case class AgentStats(mem: Double, cpu: Double, lastSeen: Instant)
+case class MesosActorConfig(agentStatsTTL: FiniteDuration)
+case class AgentStats(mem: Double, cpu: Double, ports: Int, lastSeen: Instant, expiration: Instant)
```

Comment:
are both `lastSeen` and `expiration` needed ?

Reply:
IIRC `lastSeen` can be removed ?

Reply:
I left it there in case any client may want to know "how stale" this value is, but I'm not sure of a good use case for that. Will remove it.